bitbots_bringup/launch/motion_standalone.launch

```diff
@@ -1,12 +1,14 @@
 <launch>
     <arg name="sim" default="false"/>
+    <arg name="basler" default="true"/>
```

Comment:
you could add parameter documentation here.

Reply:
Actually, there is an actual element for the XML: https://wiki.ros.org/roslaunch/XML/arg